NuRadioMC/utilities/attenuation.py

```diff
@@ -71,11 +71,19 @@ def get_attenuation_length(z, frequency, model):
             att_length_f[ att_length_f < min_length ] = min_length
 
         return att_length_f
-    elif(model == "MB1"): # from 10.3189/2015JoG14J214
+    elif(model == "MB1"): # from 10.3189/2015JoG14J214 and Phd Thesis C. Persichilli (depth dependence)
+#         from NuRadioMC.utilities import medium
+#         ice = medium.mooresbay_simple()
+
         R = 0.82
         d_ice = 576 * units.m
         att_length = 460 * units.m - 180 * units.m /units.GHz * frequency
         att_length *= (1 + att_length / (2 * d_ice) * np.log(R)) ** -1  # additional correction for reflection coefficient being less than 1. 
```

Comment:
I was unable to find where this additional correction factor is from. At least the factor 0.82 is not used in Chris' thesis. @cg-laser do you know where this is from? I know it has been in here longer, but a better documentation would be great.

Reply:
I added the missing documentation